acquia/cli
# Phar not generated for 1.7.1

**Motivation**
Fixes #437

https://github.com/composer/composer/releases/tag/2.0.10

> Fixed autoloader registration when plugins are loaded, which may impact plugins relying on this bug (if you use symfony/flex make sure you upgrade it to 1.12.2+ to fix dump-env issues)

**Proposed changes**
- Test builds as part of PRs and nightly tests to catch this sooner next time
- Run `composer update` to fix the original bug


**Testing steps**
Verify the first commit (with just the test) fails. Second commit (with composer update) should pass.

**Merge requirements**
